### PR TITLE
Release 1.4.0

### LINKED ISSUE #32: Release 1.4.0
We have a [bunch of features](https://github.com/RadicalFx/Radical.Windows/milestones/Release%201.4.0) to release and I started to think that the build failures we are facing might be due to the actual `Radical.Windows` package

### PR DESCRIPTION
**Pull request by mauroservienti**
@micdenny please review
Fix #32

### CODE REVIEW

**Comment by mauroservienti** [comment]
@micdenny ping :smile:

**Comment by micdenny** [comment]
Pong :)

File: src/net35/Radical.Windows/Presentation/Behaviors/WindowsManager.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Windows;
+using System.Windows.Interop;
+using Topics.Radical.Windows.Behaviors;
+
+namespace Topics.Radical.Windows.Presentation.Behaviors
+{
+    public class WindowsManager
```
Comment (micdenny) [inline src/net35/Radical.Windows/Presentation/Behaviors/WindowsManager.cs]: Forgot to tell that would be nice to have a class and property comment

File: src/net45/Radical.Windows/Presentation/Behaviors/WindowsManager.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Topics.Radical.Windows.Presentation.Behaviors
+{
+    class WindowsManager
+    {
```
Comment (micdenny) [inline src/net45/Radical.Windows/Presentation/Behaviors/WindowsManager.cs]: Is this class necessary?

**Comment by micdenny** [comment]
After merge I found a couple of minor stuff sorry

**Comment by mauroservienti** [comment]
there is [already a plan](https://github.com/RadicalFx/Radical.Windows/issues/38) to merge it, and I have a personal plan to enforce docs on public elements

**Comment by mauroservienti** [comment]
Any idea why was this deployed to MyGet instead of NuGet? https://ci.appveyor.com/project/radical-bot/radical-windows/deployments

**Comment by mauroservienti** [comment]
shit, my fault. Forget about it. I'll reopen a new PR

**Comment by mauroservienti** [comment]
and fix the comments as well.

**Comment by mauroservienti** [comment]
I'm force-pushing to nuget, the issue is that the [appveyor.yml](https://github.com/RadicalFx/Radical.Windows/blob/develop/appveyor.yml#L77) file is wrong. I'll fix it.

**Comment by micdenny** [comment]
I saw that the version was properly set I didn't saw other problem on appveyor file

**Comment by micdenny** [comment]
I see now that the feed was wrong :)

**Comment by mauroservienti** [comment]
yeah, the fix is in https://github.com/RadicalFx/Radical.Windows/pull/39

**Pull request merged**